docs/Other SDL Features/Encryption/index.md

```diff
@@ -79,11 +79,14 @@ ServiceEncryptionListener serviceEncryptionListener = new ServiceEncryptionListe
 !@
 
 ## Setting Optional Encryption
-If you want to encrypt a specific RPC, you must configure the payload protected status of the RPC before you send it to the head unit.
+If you want to encrypt a specific RPC, you must configure the payload protected status of the RPC before you send it to the head unit. Before sending RPCs with encryption you should call `startRPCEncryption` on the `sdlManager`.  The best place to put `startRPCEncryption` is in the successful callback of  `startWithReadyHandler`.
 
 @![iOS]
 ##### Objective-C
 ```objc
+[self.sdlManger startRPCEncryption];
```

Comment:
Also add a `TODO` comment  in the corresponding java-suite code snippet so the java-suite team knows where to update their code.